src/TipBox.vue

```diff
@@ -1,6 +1,6 @@
 <template>
     <div class="alert container" :class="[boxStyle, addClass, lightStyle]" :style="customStyle">
-        <div v-if="!isDefault" class="icon-wrapper">
+        <div v-if="!isDefault" class="icon-wrapper" :class="`${iconSize ? `fa-${iconSize}`: ''}`" >
```

Comment:
Let's do as done with `lightStyle` as suggested in https://github.com/MarkBind/vue-strap/pull/124#discussion_r372991071:

```html
<div v-if="!isDefault" class="icon-wrapper" :class="[iconStyle]">
```